MonstieWash/Assets/Scripts/Tutorial/TutorialManager.cs

```diff
@@ -28,9 +28,16 @@ private enum CompletionType { Instant, Count, Time };
     private ToolSwitcher m_toolSwitcher;
     private TaskTracker m_taskTracker;
 
+    public event Action ProgressUpdate;
+
     // scripted events - include scripted event enumtypes to be used here (tutoriial should probably have all types)
     private ScriptedEventsManager m_scriptedEventsManager;
 
+    // getters for prompt completion progress
+    public float Progress { get { return (float)Math.Round(m_trackedValue, 2, MidpointRounding.AwayFromZero); } }
+    public float Complete { get { return (float)Math.Round(m_tutorialPrompts[m_tutorialStep].Value, 2, MidpointRounding.AwayFromZero); } }
+    public string Completion { get { if (m_tutorialPrompts[m_tutorialStep].CompleteType != CompletionType.Instant) return $"{Progress} / {Complete}"; else if (m_trackedValue == -1f) return "Complete!"; else return ""; } }
```

Comment:
### ✨ **Cleanup** | _This is for keeping the codebase clean and manageable._

Just for readability, I'd want this to be split across a couple lines.

```suggestion
    public string Completion { 
        get 
            { 
                if (m_tutorialPrompts[m_tutorialStep].CompleteType != CompletionType.Instant) return $"{Progress} / {Complete}"; 
                else if (m_trackedValue == -1f) return "Complete!"; 
                else return ""; 
            } 
    }
```